pkg/labels/matcher.go

```diff
@@ -86,3 +87,16 @@ func (m *Matcher) Matches(s string) bool {
 	}
 	panic("labels.Matcher.Matches: invalid match type")
 }
+
+// openMetricsEscape is similar to the usual string escaping, but more
+// restricted. It merely replaces a new-line character with '\n', a double-quote
+// character with '\"', and a backslack with '\\', which is the escaping used by
+// OpenMetrics.
+func openMetricsEscape(s string) string {
```

Comment:
OpenMetricsEscape maybe for consistency ?

Reply:
It would export this function, showing up in the Go doc etc. I see it more as an internal helper function that I don't want to expose.